Code/GraphMol/MolPickler.cpp

```diff
@@ -461,6 +463,13 @@ void pickleQuery(std::ostream &ss, const Query<int, T const *, true> *query) {
         const auto &pval = std::get<1>(v);
         streamWrite(ss, MolPickler::QUERY_VALUE, pval);
       } break;
+      case 6: {
+        auto v =
+            boost::get<std::tuple<MolPickler::Tags, Dict::Pair, double>>(qdetails);
+        streamWrite(ss, std::get<0>(v));
```

Comment:
I should make a note that it's easier to pickle the double value (tolerance) first.